src/model/position.js

```diff
@@ -13,6 +13,9 @@ import compareArrays from '@ckeditor/ckeditor5-utils/src/comparearrays';
 import CKEditorError from '@ckeditor/ckeditor5-utils/src/ckeditorerror';
 import Text from './text';
 
+const _path = Symbol( 'path' );
+const _root = Symbol( 'root' );
```

Comment:
We don't use symbols for private properties. `@private` is enough.

Reply:
It is not really `@private`, it is read-only. Also, we do use `Symbol`s in a couple of places. I think these are fine here.

Reply:
But symbols make them private too.

Reply:
> But symbols make them private too.

A bit better hidden but also accessible ([more work required](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertySymbols) though...) also not enumerable.

But I get that in CKEditor 5 there is already established convention and using straightforward `this._path` in child classes would be simpler.

I only don like "duplication" on object as the getter `this.path` would just return `this._path`.

Reply:
Feel free to create a separate ticket to change it, but since we already do it this way, we should follow the convention:

https://github.com/ckeditor/ckeditor5-engine/blob/79da6fb1c0782cb6442527d3f2a50497b89b0985/src/model/rootelement.js#L53-L55

https://github.com/ckeditor/ckeditor5-engine/blob/79da6fb1c0782cb6442527d3f2a50497b89b0985/src/view/selection.js#L107-L119

https://github.com/ckeditor/ckeditor5-engine/blob/79da6fb1c0782cb6442527d3f2a50497b89b0985/src/view/text.js#L58-L60

Reply:
I never liked `this._data` and was wondering why it is this way. I guess I will like to change it after more important subjects are dealt with.